src/masterlist.py

```diff
@@ -80,6 +80,9 @@
     "Kameroita ei voi olla ikinä liikaa",
     "Passikuvan voi ottaa vaikka puhelimella kotona, siin ei oo kovat vaatimukset",
     "Julkaiskaa enemmän, saa palautetta ja myöhemmin saa myötistellä eiku siis näkee kuinka on kehittyny kuvaajana"
+    "Yksi konsti spiraaliongelmissa on lyödä spiraali napakasti pari kertaa pöytään. Tärähdys saa filmin siellä loppupäässä \"rentoutumaan\" ja taas kulkemaan."
```

Comment:
Laita koko rivi heittomerkkien väliin niin et tarvii pakomerkkiä lainausmerkkien eteen.

Reply:
Taas konventiot, kaikki muut merkkijonot on lainausmerkeissä ja pakomerkkejä on käytetty muuallakin, mielummin pidän tyylin yhtenäisenä.

Reply:
Hyvä näin